app/player/player.go

```diff
@@ -21,17 +21,18 @@ import (
 	"github.com/lbryio/lbry.go/v2/stream"
 )
 
-const reflectorURL = "http://blobs.lbry.io/"
+const reflectorURL = "http://blobs.lbry.io"
```

Comment:
just so you're aware, this isn't reflector, this is the actual blob storage.
reflector would be reflector.lbry.com